Fix Kops command to export k8s cluster config (#310)
cloudposse/docs: content/faq/running-kubectl-fails-connection-refused.md

This is most likely caused by not setting the `kubectl` context to use the kops cluster.

> To fix this, run `kops export kubecfg --name us-west-2.staging.cloudposse.org` (replace our kops cluster name with yours or use the `$KOPS_CLUSTER_NAME` variable, if set). =)

Reviewer: ```suggestion
To fix this, run `kops export kubecfg` (this assumes the`$KOPS_CLUSTER_NAME` variable is already set to the cluster name)
```

e.g.
```

> kops export kubecfg --name $KOPS_CLUSTER_NAME

Reviewer: ```suggestion
kops export kubecfg
```
